tbvcfreport/tbvcfreport.py

```diff
@@ -6,9 +6,12 @@
 import webbrowser
 
 import click
-
-from .report import generate_report
-from .vcfproc import VCFProc
+try:
```

Comment:
The two ways of doing relative imports are not, as far as I know, necessary. `from .report import generate_report` will work with both Python 2 and Python 3, but `from report import generate_report` will not work with Python 3.

Reply:
Had weird import issues, not sure if it was a conda/pip thing. Thought I should be cautious.